src/assets/main.css

```diff
@@ -43,33 +43,25 @@ footer {
 @media (min-width: 450px) {
   .grid-4 {
     grid-template-columns: repeat(2, 1fr);
-    grid-template-rows: auto;
-    grid-gap: 25px 25px;
   }
 }
 @media (min-width: 550px) {
   .grid-3 {
     grid-template-columns: repeat(3, 1fr);
-    grid-template-rows: auto;
-    grid-gap: 25px 25px;
   }
 }
 @media (min-width: 768px) {
   .grid-2 {
     grid-template-columns: repeat(2, 1fr);
-    grid-template-rows: auto;
-    grid-gap: 25px 25px;
   }
   .grid-4 {
     grid-template-columns: repeat(4, 1fr);
-    grid-template-rows: auto;
-    grid-gap: 25px 25px;
   }
 }
 header {
   width: 100%;
-  background-color: #708090;
-  color: #fff;
+  background-color: #30364d;
+  color: #e6e6e7;
```

Comment:
variable

Reply:
Why are you asking to set this as a variable? This is the compiled css file